pandas/core/generic.py

```diff
@@ -2276,31 +2276,30 @@ def to_json(
         orient : str
             Indication of expected JSON string format.
 
-            * Series
+            * Series:
+
```

Comment:
@datapythonista do we want the blank lines on 2280, 2283, 2285, ...?

Reply:
Not sure if we have a convention for that, but I don't think so. I think it'll be more readable without them, and markdown requires them for some implementations, but I don't think rst does.